.github/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.9"
+
+services:
+  # cd .github/workflows; docker compose -f ../docker-compose.yml up build-linux-amd64
+  build-linux-amd64:
```

Comment:
Such a winner for the local development; + mb in a separate PR this can be extended to work with GitHub Actions; 
The pattern is similar to [what we've done in the GeoTrellis repo](https://github.com/locationtech/geotrellis/blob/master/.github/workflows/ci.yml#L34-L39); the same docker-compose used for the local development and for the CI purposes; super convenient!